tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_types.ts

```diff
@@ -451,10 +446,18 @@ export interface SourceFileSpec {
 }
 
 // A specific line of a source-code file, i.e., a stack frame.
-export interface SourceLineSpec extends SourceFileSpec {
+export interface StackFrame extends SourceFileSpec {
   lineno: number;
+  function_name: string;
```

Comment:
if this is truly FE only type, use camelCase? `focusLineSpec` is camel case :\  
A counter argument can be that `SourceFileSpec` already defines snake_cased props.

Reply:
I agree that for 100% consistency, these types should be all lowerCamelCase. However, I'd like to defer it to later because of the following reasons:
1. The LOC affected by this will be large.
2. There are several other data types in this file which use snake_case in their field names. If I change the field names only for this type, it'll create some inconsistency.